README.md

```diff
@@ -52,7 +52,7 @@ Result should be:
 ### Your solution here:
 4.  Fill in the `console.log()`?
 ```js
-console.log()
+console.log(firstWord+ secondWord+ thirdWord+ fourthWord)
```

Comment:
to add a space in a string do this 

`console.log(firstWord + " " + secondWord + " " + thirdWord + " " + fourthWord)`